src/Schema/SQLServerSchemaManager.php

```diff
@@ -255,7 +263,7 @@ protected function _getPortableTableForeignKeyDefinition($tableForeignKey)
      */
     protected function _getPortableTableDefinition($table)
     {
-        if (isset($table['schema_name']) && $table['schema_name'] !== 'dbo') {
+        if ($table['schema_name'] !== 'dbo') {
```

Comment:
The `isset()` check is redundant since the columns should always be in the result set. Suppressing the notice if it doesn't exist creates a potential for regressions during the refactoring of the schema introspection queries.